dandi/tests/fixtures.py

```diff
@@ -257,6 +259,7 @@ def get_gitrepo_fixture(
     url: str,
     committish: Optional[str] = None,
     scope: Scope = "session",
+    make_subdirs_dandisets: Optional[bool] = False,
```

Comment:
`Optional` does not mean "optional argument"; it means "can be `None`," and I don't see a reason to support that here.  Get rid of the `Optional` here and below.

Reply:
ok, fixed.